src/main/java/gov/nist/oar/distrib/datapackage/DefaultDataPackager.java

```diff
@@ -127,6 +127,11 @@ public void getData(ZipOutputStream zout) throws IOException, DistributionExcept
 		    bundlelogError.append("\n Exception in getting data for: " + filepath + " at " + downloadurl);
```

Comment:
This catch block swallows the exception--that is, it doesn't rethrow after writing the messages.  Is this what you meant to do?  That is, if zout.write() fails before the file is fully written, you're going to have a zip file with a corrupted file in it.  Wouldn't it be better if you included a "throw ex;" afterward so that the whole output can be aborted?

Reply:
...or is the idea that the error log will alert the user about a corrupted file?